content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -72,105 +76,29 @@ outs:
 
 </details>
 
-## Tracking changes
-
-`dvc status` can notice when tracked data has changed (among other situations).
-To record a new version of the data, just use `dvc add` again:
-
-<details>
-
-### 👉 Expand to get an updated dataset
-
-```dvc
-$ dvc get --rev cats-dogs-v2 \
-          https://github.com/iterative/dataset-registry \
-          use-cases/cats-dogs -o datadir
-```
-
-</details>
-
-```dvc
-$ dvc status
-datadir.dvc:
-        changed outs:
-                modified:           datadir
-$ dvc add datadir
-```
-
-DVC caches the changes to the `datadir/` directory, and updates the
-`datadir.dvc` [`.dvc` file](/doc/user-guide/dvc-files-and-directories#dvc-files)
-to match the updated data. Let's commit this new version with Git:
-
-<details>
-
-### Expand to see what happened internally
-
-Use `git diff` to show the change in `datadir.dvc`:
-
-```diff
- outs:
--- md5: a304afb96060aad90176268345e10355
-+- md5: 558a00881d4a6815ba625c13e27c5b7e
-   path: datadir
-   cache: true
-```
-
-Since the contents of `datadir/` changed, its hash value is updated (to
-`558a008...`).
-
-</details>
-
-```dvc
-$ git add datadir.dvc
-$ git commit -m "Change data"
-```
-
-## Switching versions
+## Storing and sharing
 
-When we have more than one data version, we may want to switch between them. We
-can use `dvc checkout` for this. Let's say we want to revert back to the first
-`datadir/`:
+You can upload DVC-tracked data or models with `dvc push`, so they're safely
+stored [remotely](/doc/command-reference/remote). This also means they can be
+retrieved on other environments later with `dvc pull`. First, we need to setup a
+storage.
 
 ```dvc
-$ git checkout HEAD^ datadir.dvc
-$ dvc checkout datadir.dvc
+$ dvc remote add -d storage s3://my-bucket/dvc-storage
```

Comment:
```suggestion
$ dvc remote add storage s3://my-bucket/dvc-storage
```
Since the actual reproducible remote storage is in a details section below.

Reply:
nah, I think it's fine to have it in both places